src/adapter/http/src/data/verify_handler.rs

```diff
@@ -232,11 +252,15 @@ pub enum VerificationFailed {
     DatasetBlockNotFound(#[from] DatasetBlockNotFound),
 }
 
-#[derive(Debug, thiserror::Error, serde::Serialize)]
+#[derive(Debug, thiserror::Error, serde::Serialize, utoipa::ToSchema)]
 #[error("{message}")]
 pub struct OutputMismatch {
     pub message: String,
+
+    #[schema(value_type = String)]
     pub expected_hash: odf::Multihash,
```

Comment:
In future we could support `ToSchema` in odf crate directly via a feature, but I'm not chasing perfection in this PR - want to put something in place that we can polish over time.